tests/test_alchemy.py

```diff
@@ -75,6 +77,17 @@ class Meta:
     text = factory.Sequence(lambda n: "text%s" % n)
 
 
+if models.USING_POSTGRES:
+    # sqlite test database gets created/destroyed automatically, postgres does not.
+
+    def setUpModule():
```

Comment:
Really need this for the docker-compose to work properly.

Github actions creates a postgres service and mongo service for each of the workers, makes it so that there is no conflicting tests when creating/removing tables/data. It would be crazy to do that for docker-compose, so every python test service has to share the db. Meaning they need to be unique names/ and we need to create it ourselves.